mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -183,7 +187,7 @@ def handle_fallback(self, message):
     @staticmethod
     def __find_value(pods, pod_id):
         for pod in pods:
-            if pod.id == pod_id:
+            if pod_id in pod.id:
```

Comment:
It allows for more flexible searching. For example, there are multiple pod IDs from the API that correspond to a definition: Defiinition:WordData, Definition:MathData, etc. Changing this to `in` allows me to find all of them with one search rather than having cases for each.